lib/chef-dk/policyfile/dsl.rb

```diff
@@ -118,6 +118,20 @@ def cookbook(name, *version_and_source_opts)
         end
       end
 
+      def metadata
+        cookbook_root = storage_config.relative_paths_root
+        unless File.exist?(File.join(cookbook_root, "metadata.rb")) || File.exist?(File.join(cookbook_root, "metadata.json"))
+          raise PolicyfileMissingCookbookMetadata
```

Comment:
Elsewhere in this file, we catch all errors and batch them up so we can show them all at once. Does that make sense for bad metadata or is it better to just die early? Also, I forget if there is a default error catcher at a higher level that will format these errors nicely or if that only works when you stuff things into `@errors`, can you confirm that?